python/hsfs/engine/spark.py

```diff
@@ -275,3 +275,34 @@ def _setup_s3(self, storage_connector, path):
 
 class SchemaError(Exception):
     """Thrown when schemas don't match"""
+
+
+# Replace Pydoop split method to be able to support hopsfs:// schemes
+class _HopsFSPathSplitter(hdfs.path._HdfsPathSplitter):
+    import os
```

Comment:
are these imports actually needed? if yes, can they be moved to the top level?

Reply:
It doesn't look like, I think it was just a left over from the Jupyter notebook I was testing it